src/styles/blocks/snake.scss

```
@@ -0,0 +1,9 @@
+.snake {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  @media (min-width: 1200px) {
+    & {
+      grid-template-columns: repeat(3, 1fr);
+    }
```

```suggestion
      grid-template-columns: repeat(3, 1fr);
```

`@media` is applied to the block it is defined in 😉